### ISSUE TITLE
[BUG] Patrol Cats are Doubled in Sprite Window and Patrol Members List

### ISSUE THREAD

**Issue by zeldagirl113** [ClanGenOfficial/clangen#2957]
**Type**:
- Sprite - an issue with the art in the game

**Describe the bug**
When sending out a patrol with three cats, the sprites are doubled in the sprite window, showing six cats instead. The patrol members list also lists the members twice.

I'll edit this post if I find the bug again, instead of leaving tons of comments lol.

**Grade**: (only select one)
- Non-vital - This bug can easily be ignored and does not have a large impact on play

**Reproduce**
1. Start a random patrol of three cats.
2. Look at the sprite window and the patrol details.

**Commit # or Game Version Number**: e6a8cebaba4278cc28c3463c410f0fff7ca52d84

**Additional context**
N/A

**Screenshots**
not sure of patrol
![Screenshot 2024-10-25 131916](https://github.com/user-attachments/assets/02fd49e2-0cc5-47ca-b877-abed0d674608)

gen_train_argue3
![Screenshot 2024-10-25 135849](https://github.com/user-attachments/assets/210a8253-8960-4287-9bd2-ccaf054f843a)

**Comment by j-gynn** [comment]
Unable to replicate...?
![image](https://github.com/user-attachments/assets/ed7bb869-95b0-4ad2-96a9-29cdaf9065e0)

**Comment by zeldagirl113** [comment]
Yeah, I couldn't replicate it either. Must be very occasional

**Comment by zeldagirl113** [comment]
Just got it again with this patrol - mtn_bord_rogue3
![Screenshot 2024-10-25 135428](https://github.com/user-attachments/assets/427e9205-dd6b-483f-9f9c-06e196dc9e72)

**Comment by tobes03** [comment]
I've noticed this one multiple times when playing dev, too. I've gotten it on 2 cat patrols. I have a hunch it has something to do with selecting cats by double-clicking really fast, but I haven't verified it. I'll add screenshots when it comes up again.

**Comment by scribblecrumb** [comment]
I have a feeling that #2962 is being caused by the same thing

**Comment by PoppyBlossom** [comment]
Okay, guys, this has definitely been caused by double-clicking too fast, I replicated it by using an auto-clicker. Multiple other issues are caused by the same thing. I think all of this was caused by #2577 (?). 

Also, the number of times you click the patrol button will decide how many duplicate cats there are.
![image](https://github.com/user-attachments/assets/984721f3-3e9c-42f1-b335-f6d9686306e9)
